pennylane/ops/qutrit/channel.py

```diff
@@ -222,3 +222,93 @@ def compute_kraus_matrices(p):  # pylint:disable=arguments-differ
         )
 
         return [identity] + Ks
+
+
+class QutritAmplitudeDamping(Channel):
+    r"""
+    Single-qutrit amplitude damping error channel.
```

Comment:
I wonder why it shows up like this in the docs:
![image](https://github.com/PennyLaneAI/pennylane/assets/49409390/a2768eb4-ab2d-4cdd-9013-457098929b82)

We should be able to manually override that though, e.g., we [do it here](https://github.com/PennyLaneAI/pennylane/blob/59c32874dc825af65a86026ea51f68b9338446a4/pennylane/ops/qubit/non_parametric_ops.py#L35).

Reply:
@trbromley  I can do that, but it seems like there may be a bigger issue. I just checked other operators and they all seem to be doing this. For example this behavior is happening in [PR 5676 for the RX gate](https://xanaduai-pennylane--5668.com.readthedocs.build/en/5668/code/api/pennylane.RX.html). This is different than the stable version. I tracked back the first commit who's PRs docs has this occur and I found that [PR 5511](https://xanaduai-pennylane--5511.com.readthedocs.build/en/5511/code/api/pennylane.RX.html) seems to be the start. I am just using the docs built by CI so it's likely I'm wrong about which PR is the first, but I think that there is a bug here. I'd be happy to do some more exploration and build the docs myself for each commit to make a bug report if that would help.

Reply:
Update: I tested the docs on my machine for the commit where #5511 is added and found that it had the same behavior as in the CI generated one.
I also tested the docs locally for the commit for #5595 the PR that was merged directly before #5511. In this commit the docs had the expected values in the arguments for functions, ie they were generated properly.

Note: This only seems to be happening for docs of Operator class types. This is consistent with what was changed in 5511.

Reply:
I think this is out of scope for this PR, if there is a deeper underlying problem, we can make an issue for it and then reference it in a comment in this discussion thread so the source can be easily found.

Reply:
I have made a bug report #5724.